docs/hub/spaces-overview.md

```diff
@@ -131,3 +131,19 @@ Select "Clone repository", and then you'll be able to follow the instructions to
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone2.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone1.png"/>
 </div>
+
+## Linking Models on the Hub
+
+You can showcase all the models and datasets that your Space links to by adding their identifier in your Space's README metadata. To do so, you can define them under the `models` and `datasets` keys. For example:
+
+```
+title: My lovely space
+emoji: 🤗
+colorFrom: blue
+colorTo: green
+sdk: docker
+pinned: false
+models:
+- reach-vb/musicgen-large-fp16-endpoint
+```
+
```

Comment:
Would you mind also adding something along the lines of the second point in https://huggingface.slack.com/archives/C02EMARJ65P/p1693213871117809?thread_ts=1693042670.654759&cid=C02EMARJ65P please?

Reply:
Good point! Done!